procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -299,8 +299,8 @@ Merge central to beta
 Re-opening the tree(s)
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Ask Sheriffs and RelMan to re-open trees (either ``open`` or
-``approval-only``) so that **l10n bumper can run**.
+`Reopen the mozilla-beta tree <https://treestatus.mozilla-releng.net/static/ui/treestatus/show/mozilla-beta>`__
+by restoring the previous state so that **l10n bumper can run**.
```

Comment:
As Aki suggested yesterday, long-term solution is to have the l10n bumper be part of the behavior and submit that without `DONTBUILD` and with `CLOSE TREES`. Tracked in https://bugzilla.mozilla.org/show_bug.cgi?id=1666321. Until then, thanks for fixing the docs!